resources/views/books/list-item.blade.php

```diff
@@ -1,10 +1,10 @@
 <div class="book entity-list-item"  data-entity-type="book" data-entity-id="{{$book->id}}">
-    <h4 class="text-book"><a class="text-book entity-list-item-link" href="{{$book->getUrl()}}"><i class="zmdi zmdi-book"></i><span class="entity-list-item-name">{{$book->name}}</span></a></h4>
+    <h4 class="text-book"><a class="text-book entity-list-item-link" href="{{$book->getUrl()}}"><i class="zmdi zmdi-book"></i><span class="entity-list-item-name">{{$book->name}}</span><small class="small-title">{{ trans('entities.book') }}</small></a></h4>
```

Comment:
Not sure if this will be merged since in BookStack generally icons and colors are used to distinguish between entities.

Reply:
But it looks ambiguous!